pkg/apis/serving/v1alpha1/configuration_types.go

```diff
@@ -95,7 +115,7 @@ type ConfigurationCondition struct {
 	Status corev1.ConditionStatus `json:"status" description:"status of the condition, one of True, False, Unknown"`
 
 	// +optional
-	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
+	LastTransitionTime VolatileTime `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
```

Comment:
Add some comment explaining why we choose `VolatileTime` vs `metav1.Time` ?

Reply:
Done.